packages/autocomplete-plugin-tags/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export type { Tag } from './types';
```

Comment:
We can export everything from `'./types'`, like we're doing in the other packages.

Reply:
It would export `BaseTag` as well. Do we want that?

Reply:
We export `BaseItem` which is necessary when developing plugins user-land, so I think it can be useful for `BaseTag` too.

Reply:
Addressed in 57be5f1a.